AsyncDisplayKit/ASTextNode.mm

```diff
@@ -250,9 +248,28 @@ - (ASTextKitAttributes)_rendererAttributes
   };
```

Comment:
It might be worth adding a comment indicating that the method is assumed to have the lock held.